src/components/VocabularyList.tsx

```
@@ -106,11 +109,44 @@ const VocabularyList = ({
+        .eq("id", vocabularyId);
+
+      if (error) {
+        throw error;
+      }
+    },
+    onSuccess: () => {
+      queryClient.invalidateQueries({
+        queryKey: ["vocabulariesWithStories", user?.id],
+      });
+    },
+    onError: (error: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)